js/cryptocom.js

```diff
@@ -1422,9 +1423,16 @@ module.exports = class cryptocom extends Exchange {
     }
 
     safeNetwork (networkId) {
-        // stub for now
-        // TODO: figure out which networks are supported on cryptocom
-        return networkId;
+        const networksById = {
+            'BTC': 'BTC',
```

Comment:
question, what's the benefit of having this map if we're going to map X to X? We're using the same id for the key and value

Reply:
Do we have unify network name, eg ERC20?

I was thinking to add `ERC20: ETH` or `BEP20: BNB` in safeNetwork, and we can use that when user call withdraw.

Reply:
@sc0Vu I think the network unification is not "complete" yet, but yes we usually convert network ids to ERC20, TRC20, BSC,OMNI,SOL,BCH, etc (you can check other `safeNetwork` Implementations for more examples).

My point is if the name is the same we don't need to list it right? Just the different ones

Reply:
For now it's only for clarity, we can leave this PR till  the network unification is completed.